lesson06/assignment/Drama.html

```diff
@@ -0,0 +1,114 @@
+                        <a class="nav-link" href="comedyshows.html">Comedy shows</a>
+                    </li>
+                    <li class="nav-item">
+                        <a class="nav-link" href="kids.html">Kids</a>
+                    </li>
+                </ul>
+            </div>
+        </nav>
+  
+            <section>
+                    <h3>Filters</h3>
+                    <nav>
+                        <ul>
+                          <li>Language
+                            <ul class="menu-1">
+                              <li class="menu-1__li">English</li>
+                              <li>Spanish</li>
+                              <li>French</li>
+                              <li>Japanese</li>
+                              <li>Korean</li>
+                            </ul>
+                          </li>
+                        </ul>
+                      </nav>
+                    </section>
+                    <aside>
+                
+                <!--responsive/fluid bootstrap images-->
+
+                <div class="row1">
+                    <div class="item-1">
+                        <img class="img-fluid img-thumbnail" src="images/10.jpg" alt="event one">
+                        <p class="event-name">Halloween<br>Seattle</p>
+                    </div>
+                    <div class="item-1">
+                        <img class="img-fluid img-thumbnail" src="images/3.jpg" alt="event one"> 
+                        <p class="event-name">Party Night<br>Bellevue</p>
+                    </div>
+                </div>
+                <div class="row1">
+                    <div class="item-1">
+                        <img class="img-fluid img-thumbnail" src="images/8.jpg" alt="event one">
+                        <p class="event-name">Alice in Wonderland<br>Tacoma</p>
+                    </div>
+                    <div class="item-1">
+                        <img class="img-fluid img-thumbnail" src="images/7.jpg" alt="event one">
+                        <p class="event-name">Pokemon<br>Seattle</p>
+                    </div>
+                </div>
+                <div class="row1">
+                    <div class="item-1">
+                        <img class="img-fluid img-thumbnail" src="images/11.jpg" alt="event one">
+                        <p class="event-name">Bass Monsters<br>Tacoma</p>
+                    </div>
+                    <div class="item-1">
+                        <img class="img-fluid img-thumbnail" src="images/5.jpg" alt="event one">
+                        <p class="event-name">Salsa at Alki beach<br>Seattle</p>
+                    </div>
+                </div>
+            </div>
+        </aside>
+    </body>
+        <footer>
+            <p class="footer-text"><strong>Book My Event</strong> <br>Seattle, Bellevue and Tacoma <br>111 Pike place market, Seattle, WA <br><small>All rights reserved</small></p>
+        </footer>
+    </div>
+</header>
```

Comment:
This should be closed way up above where the header ends.  You don't want your entire page of markup inside this header tag.